From a3e35565277be2aa5809e1fc699cab8782e56dbb Mon Sep 17 00:00:00 2001
From: Vidya <vvelumuri@marvell.com>
Date: Tue, 24 Nov 2020 06:25:09 +0000
Subject: [PATCH 1051/1921] crypto: cn10k: update the opcode for inline inbound
 ipsec

The opcode for inline inbound ipsec is changed for cn10k.
Update the macro with the updated value of the opcode.

Signed-off-by: Vidya <vvelumuri@marvell.com>
Change-Id: Ic78b9f3be4d5e93babdd58c22b92a5e79f074b83
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42416
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c b/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c
index 49c711931833..3d309a33f72d 100644
--- a/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c
+++ b/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c
@@ -5,7 +5,7 @@
 #include "rvu_reg.h"
 
 /* Fastpath ipsec opcode with inplace processing */
-#define CPT_INLINE_RX_OPCODE (0x26 | (1 << 6))
+#define CPT_INLINE_RX_OPCODE (0x29 | (1 << 6))
 /*
  * CPT PF driver version, It will be incremented by 1 for every feature
  * addition in CPT PF driver.
-- 
2.31.1

